contracts/contracts/strategies/MorphoCompoundStrategy.sol

```diff
@@ -0,0 +1,247 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity ^0.8.0;
+
+/**
+ * @title OUSD Morpho Compound Strategy
+ * @notice Investment strategy for investing stablecoins via Morpho (Compound)
+ * @author Origin Protocol Inc
+ */
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { BaseCompoundStrategy } from "./BaseCompoundStrategy.sol";
+import { IERC20 } from "../utils/InitializableAbstractStrategy.sol";
+import { IMorpho } from "../interfaces/morpho/IMorpho.sol";
+import { ILens } from "../interfaces/morpho/ILens.sol";
+import { StableMath } from "../utils/StableMath.sol";
+import "../utils/Helpers.sol";
+
+contract MorphoCompoundStrategy is BaseCompoundStrategy {
+    address public constant MORPHO = 0x8888882f8f843896699869179fB6E4f7e3B58888;
+    address public constant LENS = 0x930f1b46e1D081Ec1524efD95752bE3eCe51EF67;
+    using SafeERC20 for IERC20;
+    using StableMath for uint256;
+
+    /**
+     * @dev Initialize function, to set up initial internal state
+     * @param _vaultAddress Address of the Vault
+     * @param _rewardTokenAddresses Address of reward token for platform
+     * @param _assets Addresses of initial supported assets
+     * @param _pTokens Platform Token corresponding addresses
+     */
+    function initialize(
+        address _vaultAddress,
```

Comment:
nit: Since we use a proxy for vault, could hardcode that address too, just like the Morpho and Lens contract addresses above

Reply:
We have vault configurable by all of our strategies on initialisation. I would rather leave it as is for the sake of consistency. But I do agree with you. 

We probably needed it to be configurable in the past because of unit tests (using a mock Vault there). New strategies mostly utilise fork tests going forward. It might be nice for the strategy to be able to support the unit test environment just in case we want to write some in the future.

Reply:
Agree on keeping it configurable for now.